src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java

```diff
@@ -77,6 +77,10 @@ protected DeviceData parseDeviceData(DocumentContext document) throws ParseExcep
             for (TransformerKVMapping mapping : mappings) {
                 String key = eval(document, mapping.getKey());
                 String strVal = eval(document, mapping.getValue());
+                if(mapping.getValue().equals(strVal)){
+                    //取值错误，跳过这个数据
```

Comment:
Please provide comment in English or remove it

Reply:
Revised